behat.yml

```diff
@@ -0,0 +1,32 @@
+# Run admin behat tests with this command (installed with silverstripe/installer)
+# Note that admin behat tests require CMS module
+# ========================================================================= #
+# vendor/bin/selenium-server-standalone -Dwebdriver.firefox.bin="/Applications/Firefox31.app/Contents/MacOS/firefox-bin"
+# vendor/bin/serve --bootstrap-file vendor/silverstripe/admin/tests/behat/serve-bootstrap.php
+# vendor/bin/behat @admin
+# ========================================================================= #
```

Comment:
These comments were obviously copied from `silverstripe/silverstripe-admin`... please either update them, or remove them if they're not relevant.